tauri/src/endpoints/global_shortcut.rs

```diff
@@ -0,0 +1,86 @@
+use crate::{api::shortcuts::ShortcutManager, async_runtime::Mutex};
+use once_cell::sync::Lazy;
+use serde::Deserialize;
+
+use std::sync::Arc;
+
+type ShortcutManagerHandle = Arc<Mutex<ShortcutManager>>;
+
+pub fn manager_handle() -> &'static ShortcutManagerHandle {
+  static MANAGER: Lazy<ShortcutManagerHandle> = Lazy::new(Default::default);
+  &MANAGER
+}
+
+/// The API descriptor.
+#[derive(Deserialize)]
+#[serde(tag = "cmd", rename_all = "camelCase")]
+pub enum Cmd {
+  /// Register a global shortcut.
+  Register {
+    shortcut: String,
+    handler: String,
+    callback: String,
+    error: String,
+  },
+  /// Unregister a global shortcut.
+  Unregister {
+    shortcut: String,
+    callback: String,
+    error: String,
+  },
+}
+
+impl Cmd {
+  #[allow(unused_variables)]
```

Comment:
Why is this here?

Reply:
hmm I rewrote the method and it isn't needed anymore; thanks for finding that